src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -631,4 +631,7 @@ For details on valid range queries, please refer to Search Matching section in C
     <value>Invalid query: specified tag '{0}' value '{1}' contains unsupported character '{2}'. Character is not supported with Fuzzy match, try exact match or removing the invalid character. </value>
     <comment>{0} Dicom Tag name, {1} Tag value to search. {2} Unsupported character.</comment>
   </data>
+  <data name="DatasetDoesNotMatchSOPClass" xml:space="preserve">
+    <value>Dataset does not match SOP Class.</value>
```

Comment:
Is there not an existing resource string for this?

Reply:
I couldn't find one that matches this value literal.